anyTV/freedom-crypto-miner-i18n
# VN-495 toggle to mine even projected profit loss

## Summary of changes

* add toggle in Mining setting page to let user decide mine event projected profit loss
* get miner logic was changed to
- Manual -> get crypto from preferred list -> filter based on toggle mining option above -> select miner
- Auto -> get minable cryptoes -> filter based on toggle mining option above -> select miner

<img width="582" alt="untitled 2" src="https://user-images.githubusercontent.com/8542805/48465137-623db000-e814-11e8-8ede-15fb044f3be9.png">

## Checklist


## Testing

- Default mine event projected profit loss is off
- Change wattage to got not profitable estimation -> got notify when start miner / auto mine idle
![image](https://user-images.githubusercontent.com/8542805/48464991-f78c7480-e813-11e8-8ddc-c3e1684fb5e4.png)

- Enable "Mine even projected profit loss" -> make sure start manual / auto mine idle can select miner for mining
